crates/metadata/src/specs.rs

```diff
@@ -81,8 +63,8 @@ impl IntoPortable for ContractSpec {
                 .into_iter()
                 .map(|event| event.into_portable(registry))
                 .collect::<Vec<_>>(),
-            docs: registry.map_into_portable(self.docs),
-            lang_error: registry.register_type(&self.lang_error),
+            docs: self.docs.into_iter().map(|s| s.into()).collect(),
```

Comment:
@xermicus was this change required?

Reply:
I don't think so, this looks more like an oversight on my side, sorry.